Sources/ApplicativeRouter/RequestData.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+import Prelude
+
+struct RequestData: Monoid {
```

Comment:
this is just the value we parse from and print into. the parser-printer works over any monoid.